style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  background-image: url(/images/header-shapes\ mobile.png);
```

Comment:
- [ ] Background image also not working. Make sure every image is working properly.